package/piksi_system_daemon/piksi_system_daemon/src/cellmodem_probe.c

```diff
@@ -167,7 +167,7 @@ enum modem_type cellmodem_probe(const char *dev, sbp_zmq_pubsub_ctx_t *pubsub_ct
       return MODEM_TYPE_INVALID;
     }
     piksi_log(LOG_INFO, "Modem %s: %s", c->display, r);
```

Comment:
FYI, I added a LOG_SBP flag to the loglevel parameter to handle these cases, so you can do the following to syslog and SBP:

      piksi_log(LOG_INFO|LOG_SBP, "Modem %s: %s", c->display, r);

Reply:
Cool yea I saw that, is that in 1.5 too?